src/main/java/io/harness/cf/client/api/CfClient.java

```diff
@@ -158,7 +161,11 @@ public boolean boolVariation(String key, Target target, boolean defaultValue) {
       return defaultValue;
     } finally {
       if (!target.isPrivate() && isAnalyticsEnabled) {
-        analyticsManager.pushToQueue(target, featureConfig, servedVariation);
+        if (analyticsManager != null) {
```

Comment:
Its possible for AnalyticsManager to be null in a number of scenarios.  This can cause a null pointer exception unless we handle it.   We can just log here, though the problem is analyticsManager wont necessarily get initialised after startup, so it maybe better to check for this in the constructor and return an error to the calling code.